src/main/java/jenkins/plugins/zulip/DescriptorImpl.java

```diff
@@ -94,6 +96,22 @@ public void setTopic(String topic) {
         this.topic = topic;
     }
 
+    public boolean isFullJobPathAsDefaultTopic() {
```

Comment:
You also need to add the new fields to the `configure` method:

fullJobPathAsDefaultTopic = Boolean.TRUE.equals(json.get("fullJobPathAsDefaultTopic"));
fullJobPathInMessage = Boolean.TRUE.equals(json.get("fullJobPathInMessage"));